frontend/packages/noobaa-storage-plugin/src/components/object-data-reduction-card/object-data-reduction-card-item.tsx

```diff
@@ -50,13 +50,14 @@ export const SavingsItem: React.FC<SavingsItemProps> = React.memo(
       text = <LoadingInline />;
     } else if (!_.isNil(savings)) {
       const savingsPercentage = logicalSize
-        ? humanizePercentage((100 * Number(savings)) / Number(logicalSize))
+        ? `(${humanizePercentage((100 * Number(savings)) / logicalSize).string})`
         : null;
       const savingsFormattted = humanizeBinaryBytesWithoutB(Number(savings));
       text = (
         <span className="nb-object-data-reduction-card__row-status-item-text">
           {savingsFormattted.value}
-          {savingsFormattted.unit}({savingsPercentage.string})
+          {savingsFormattted.unit}
```

Comment:
you should use `savingsFormattted.string` instead of `value` and `unit` as `string` property adds space between unit and value per unit definition.

also it seems to me that `savingsFormattted` and `savingsPercentage` is going to be shown right next to each other without any space ? like `20Ki10%` ?

Reply:
@rawagner yes as per the mock https://projects.invisionapp.com/share/BRS4A5ETN3H#/screens/364134659_Object_storage_Dashboard-_IO_operations_for_providers
its together without space

Reply:
made it spacious @rawagner so as to be consistent with overall console 
![per](https://user-images.githubusercontent.com/25664409/62553416-eb0de780-b88c-11e9-9b6f-e1718ae8304c.png)